v2/codetags/parser.go

```diff
@@ -324,19 +350,59 @@ parseLoop:
 		case stMaybeValue:
 			switch {
 			case r == '=':
+				s.next()
+				hasValue = true
```

Comment:
Same as elsewhere: should this set incomplete? If you hit EOF right after = it should be an error, right?

Reply:
Fixed and added test.  For ParseRaw values it remains allowed.